src/runes/runestone.rs

```diff
@@ -0,0 +1,501 @@
+use super::*;
```

Comment:
Would be awesome to have runestone as its own crate so you can depend on it without all of ord

Reply:
I'm definitely open to pulling it out later.